packages/gatsby-plugin-now/gatsby-node.js

```diff
@@ -0,0 +1,33 @@
+const path = require("path");
+const writeFile = require("util").promisify(require("fs").writeFile);
+
+const REDIRECT_FILE_NAME = "__now_routes.json";
+
+exports.onPostBuild = async ({ store }) => {
+  const { redirects, program } = store.getState();
+
+  if (!redirects.length === 0) {
+    return;
+  }
+
+  const routes = [{ handle: 'filesystem' }]
+
+  for (let redirect of redirects) {
+    let route = {
+      src: redirect.fromPath,
+      status: redirect.statusCode || (redirect.isPermanent ? 301 : 302),
+      headers: { Location: redirect.toPath }
+    };
+
+    if(redirect.force) {
+      routes.unshift(route)
+    } else {
+      routes.push(route);
+    }
+  }
+
+  await writeFile(
+    path.join(program.directory, "public", REDIRECT_FILE_NAME),
```

Comment:
Is `public` correct here? I think it will get deployed as a file when it gets in there, maybe it should just be in `program.directory`?

Reply:
I chose to put in `public` because when you run `gatsby build` locally, the file will be generated but we don't want users to push it to their version control system.

I can chose a name that's less likely to conflict and delete it in `@now/static-build` to avoid deploying it.

Reply:
Right. Deleting it in a different package seems a bit unintuitive though, but I can't think of something better.

Maybe we could add a timestamp to the file and then search for the file in `@now/static-build` and then delete it.

Reply:
Why a timestamp @AndyBitz?

Reply:
@lucleray We could use the sha sum as well, but then we had to read the file etc. I thought of something like this `__now_redirects.<timestamp>.json`.

Reply:
Just to make sure it is not a fixed filename